ketcher-autotests/tests/API/view-only-mode.spec.ts

```diff
@@ -0,0 +1,544 @@
+    page,
+  }) => {
+    /*
+    Test case: https://github.com/epam/ketcher/issues/4965
+    Description: All editing tools are disabled in toolbars for elements
+    */
+    await openFileAndAddToCanvasAsNewProject(
+      'KET/images-png-50-with-50-structures.ket',
+      page,
+    );
+    await enableViewOnlyModeBySetOptions(page);
+    await takePageScreenshot(page);
+  });
+
+  test('Turn on view-only mode, add to canvas different elements from file, verify that all editing tools are disabled in toolbars for elements', async ({
+    page,
+  }) => {
+    /*
+    Test case: https://github.com/epam/ketcher/issues/4965
+    Description: Turn on view-only mode, add to canvas different elements from file, verify that all editing tools are disabled in toolbars for elements
+    */
+    await enableViewOnlyModeBySetOptions(page);
+    await openFileAndAddToCanvasAsNewProject(
+      'KET/images-png-50-with-50-structures.ket',
+      page,
+    );
+    await takePageScreenshot(page);
+  });
+
+  test('Verify that the following tools and functions are enabled in view-only mode(Open, Save, Copy) ', async ({
+    page,
+  }) => {
+    /*
+    Test case: https://github.com/epam/ketcher/issues/4965
+    Description: Tools and functions are enabled in view-only mode(Open, Save, Copy) 
+    */
+    await selectRingButton(RingButton.Benzene, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await enableViewOnlyModeBySetOptions(page);
+    await page.keyboard.press('Control+a');
+    await expect(page.getByTestId('open-file-button')).toBeEnabled();
+    await expect(page.getByTestId('save-file-button')).toBeEnabled();
+    await expect(page.getByTitle('Copy (Ctrl+C)')).toBeEnabled();
+    await takeTopToolbarScreenshot(page);
+  });
+
+  test('Verify that the hand and selection tools are enabled in view-only mode', async ({
+    page,
+  }) => {
+    /*
+    Test case: https://github.com/epam/ketcher/issues/4965
+    Description: Hand and selection tools are enabled in view-only mode
+    */
+    await enableViewOnlyModeBySetOptions(page);
+    await expect(page.getByTestId('hand')).toBeEnabled();
+    await expect(page.getByTestId('select-rectangle-in-toolbar')).toBeEnabled();
+    await takeLeftToolbarScreenshot(page);
+  });
+
+  test('Verify that elements on Canvas can be copied (as MOL) in view-only mode', async ({
+    page,
+  }) => {
+    /*
+    Test case: https://github.com/epam/ketcher/issues/4965
+    Description: Elements on Canvas copied (as MOL) in view-only mode
+    */
+    await selectRingButton(RingButton.Benzene, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await enableViewOnlyModeBySetOptions(page);
+    await page.keyboard.press('Control+a');
+    await page.getByTestId('copy-button-dropdown-triangle').click();
+    await page.getByTitle('Copy as MOL (Ctrl+M)').click();
+    await disableViewOnlyModeBySetOptions(page);
+    await page.keyboard.press('Control+v');
```

Comment:
Better to use pasteFromClipboardByKeyboard method - since copy is asynchronous function and could lasts long